datastore/item_count.go

```diff
@@ -49,7 +51,7 @@ func (counts *ClientItemCounts) SumHistoryCounts() int {
 
 func (dynamo *Dynamo) initRealCountsAndUpdateHistoryCounts(counts *ClientItemCounts) error {
 	now := time.Now().Unix()
-	if counts.LastPeriodChangeTime == 0 {
+	if counts.Version < currentCountVersion {
```

Comment:
If this field is new (`.Version`), does it default to 0 / able to do comparison? Expecting it should evaluate as true - then it's setting to `currentCountVersion` (2). Then next time, you might manually increase `currentCountVersion` to 3 (if something changes about how we do count)?